submissions/rafiatAminu/HTML/productPage.html

```diff
@@ -11,8 +11,8 @@
     <h1>RAF PHONE STORES</h1>
     <nav>
         <ul>
-            <li><a href="#">Login</a></li>
-            <li><a href="#">Contact</a></li>
+            <li><a href="./productPage.html">Store</a></li>
```

Comment:
Adding of navigation links will be done in your next task. Kindly remove it for now